### ISSUE TITLE
Connectivity from DOM1 to MP LAA networks

### ISSUE THREAD

**Issue by davidkelliott** [ministryofjustice/modernisation-platform#4622]
## User Story

User request, see thread here - https://mojdt.slack.com/archives/C01A7QK5VM1/p1689938702369069

Connectivity is needed from DOM1 through to LAA VPCs in MP on port 443.

This is to allow end users to access an application in the `laa-ccms-infra-azure-ad-sso` account, a "Vision" instance of Oracle EBS.

Contact: Keith Humphreys

## User Type(s)

MP User

## Value

Enable end users to access the CCMS Vision instance.

### Proposal

Investigate and put into place/advise of changes needed.
Potential changes required:
 - MP firewall
 - MoJ WAN networking
 - DOM1 PAC file changes

## Reference

[How to write good user stories](https://www.gov.uk/service-manual/agile-delivery/writing-user-stories)

**Comment by davidkelliott** [comment]
As part of the PPUD migration connectivity between sites / DOM1 RAS and MP has been enabled. As far as the PAC file is concerned there are several CCMS related entries that cause the traffic to go direct. If the URLs that you are using are different to the ones already n the PAC file you will need to get Atos to add them, if not it should now work. The proxy migration work won't affect this as the entries will already be direct.
In a nutshell, both traffic from sites and DOM1 RAS traffic destined for MP is routed to the Ark DC's where they are sent to the Palo Alto firewalls, which then send them via a VPN to MP. The PAC file entry is needed to stop the traffic being sent to the Forcepoint proxies.
Once the proxy migration is complete there will be no PAC file.

**Comment by SimonPPledger** [comment]
Requires a PAC file change - and therefore we need to raise an RFC. I am talking to Katherine Miller 24/8 to fill this in and will get David and Dave to review.
We need to think about how it is implemented.
The plan is for it to cover all mod platform access from dom1/mojo

**Comment by SimonPPledger** [comment]
Waiting for confirmation of the budget holder. Will then submit the form

**Comment by SimonPPledger** [comment]
Have submitted form now - waiting for confirmation of cost and then for a scheduled date for implementing.

**Comment by SimonPPledger** [comment]
DMND0003123 PAC file changes to allow direct access to Modernisation Platform from MoJ Devices
" have allocated this to Dawn Candeland who will be in touch within the next 3 working days to discuss the next steps with you."

**Comment by davidkelliott** [comment]
Completed and tested
